### PR TITLE
WLSSB: Newest Auth Updates and add LU

### PR DESCRIPTION
**Pull request by WorseThanDeath**

### CODE REVIEW

**Comment by Volco** [comment]
@Lycanium-Z this is tested correct (only really 1 thing to test lol the slight move change)

**Comment by Lycanium-Z** [comment]
@Volco @HoeenCoder tested

**Comment by Volco** [comment]
@Lycanium-Z the travis needs fixing lol

**Comment by Lycanium-Z** [comment]
fuck

**Comment by Lycanium-Z** [comment]
ok im stuck :/ 

also Digimon Showdown is a bigger priority. So this might be untouched for a bit

File: config/formats.js [hunk 1]
```diff
@@ -568,6 +568,15 @@ exports.Formats = [
 		mod: 'oldweather',
 		ruleset: ['[Gen 7] OU'],
 	},
+	{
+		name: "[Gen 7] LU",
+		desc: ["The new tiering format that nobody wanted! This format is much more chaotic compared to OU and will be tiered differently. Ran by Lycanium Z. "],
+		column: 2,
+
+		mod: 'lu',
+		ruleset: ['Pokemon', 'standardlu', 'Team Preview'],
```
Comment (Volco) [inline config/formats.js]: ``'standardlu'`` should be ``'Standard LU'``

**Comment by Volco** [comment]
now to just fix travis

**Comment by WorseThanDeath** [comment]
ok then :|

**Comment by WorseThanDeath** [comment]
ready for merge and tested! 🍰

File: config/formats.js [hunk 2]
```diff
@@ -568,6 +568,15 @@ exports.Formats = [
 		mod: 'oldweather',
 		ruleset: ['[Gen 7] OU'],
 	},
+	{
+		name: "[Gen 7] LU",
+		desc: ["The new tiering format that nobody wanted! This format is much more chaotic compared to OU and will be tiered differently. Ran by Lycanium Z. "],
```
Comment (Volco) [inline config/formats.js]: Shouldn't "Ran by Lycanium Z" say ~~"ran by the guy with too many names"~~ i mean "Ran by Worse Than Death"

Comment (Volco) [reply config/formats.js]: @WorseThanDeath pinging you so you see this message

Comment (WorseThanDeath) [reply config/formats.js]: Yeah, I’m moving my 
 account to a new email HECKIN spam

File: config/formats.js [hunk 3]
```diff
@@ -553,7 +553,7 @@ exports.Formats = [
 	{
 		name: "[Gen 7] Stacked",
 		desc: [
-			"All hazards can be stackable and stacked infinitely. Original idea by Serperiorater. Coded by Lycanium Z.",
+			"All hazards can be stackable and stacked infinitely. Original idea by Serperiorater. Coded by Worse than Dath.",
```
Comment (Volco) [inline config/formats.js]: ~~go home lycan you're drunk~~ *Death

**Pull request merged**